src/test/resources/features/component.feature

```diff
@@ -0,0 +1,6 @@
+Feature: Messaging Listeners and Senders
+
+  Scenario: Persist stat update request and verify persistence
+    Given a stat-update-request payload "{\"trainerId\":\"5\", \"year\":\"2025\", \"month\":\"8\", \"duration\":\"60\", \"actionType\":\"ADD\", \"userName\":\"Timmy.Smith\", \"firstName\":\"Timmy\", \"lastName\":\"Smith\", \"status\":\"true\"}"

```

Comment:
payload can be moved to the separate file

Reply:
added a separate payload.json file